tree.go

```diff
@@ -255,4 +255,5 @@ var (
 	gbr     = newMIME("image/x-gimp-gbr", ".gbr", magic.Gbr)
 	xfdf    = newMIME("application/vnd.adobe.xfdf", ".xfdf", magic.Xfdf)
 	glb     = newMIME("model/gltf-binary", ".glb", magic.Glb)
+	jxr     = newMIME("image/vnd.ms-photo", ".jxr", magic.Jxr)
```

Comment:
Please use `image/jxr` as main MIME type and `image/vnd.ms-photo` as alias.
The reason is because `image/jxr` is the official MIME type registered at [IANA](https://www.iana.org/assignments/media-types/image/jxr).